kotlin/workflow-ui-android/src/main/java/com/squareup/workflow/ui/ViewRegistry.kt

```diff
@@ -86,8 +87,14 @@ class ViewRegistry private constructor(
   /**
    * Creates a [View] to display [initialRendering], which can be updated via calls
    * to [View.showRendering].
+   *
+   * @throws IllegalArgumentException if no binding can be find for type [RenderingT]
+   *
+   * @throws IllegalStateException if the matching [ViewBinding] fails to call
+   * [View.bindShowRendering] when constructing the view
    */
-  fun <RenderingT : Any> buildView(
+  @PublishedApi
+  internal inline fun <reified RenderingT : Any> buildView(
```

Comment:
This method body is non-trivial, could we just pass a `KClass` to it and make a reified inline wrapper?

Reply:
I suppressed a couple of warnings and was able to get rid of all the inlining.